opentech/apply/funds/templates/funds/application_base_landing.html

```diff
@@ -2,11 +2,11 @@
 {% block header_modifier %}header--light-bg{% endblock %}
 {% block content %}
 <div class="wrapper wrapper--small">
-    <h3>Thank you for your submission to the Open Technology Fund.</h3>
+    <h3>Thank you for your submission to the {{ ORG_LONG_NAME }}.</h3>
     <div class="rich-text">
         <p>An e-mail with more information has been sent to the address you entered.</p>
         <p>If you do not receive an e-mail within 15 minutes please check your
-spam folder and contact help@opentech.fund for further assistance.</p>
+spam folder and contact {{ ORG_EMAIL }} for further assistance.</p>
```

Comment:
@danblah In most places a "hello" address is used but here a "help" address is used. Can we simplify this to a single "ORG_EMAIL" or do we need to have a "ORG_SUPPORT_EMAIL" as well?

Reply:
yes